### PR TITLE
Move goth integration tests to yagna repo

### PR DESCRIPTION
**Pull request by zakaprov**
Resolves: https://github.com/golemfactory/goth/issues/401

### CODE REVIEW

File: goth_tests/README.md
Comment (azawlocki) [inline goth_tests/README.md:38]: One may wonder if this is required for the test framework.

File: goth_tests/README.md
Comment (azawlocki) [inline goth_tests/README.md:123]: Better to compare `poetry run poe goth-tests` with `poetry run pytest -svx e2/vm/test_e2e_vm.py` here (or `poe goth-tests` with `pytest -svx e2e/vm/test_e2e_vm.py`, assuming the appropriate virtual env is activated).

File: goth_tests/README.md
Comment (azawlocki) [inline goth_tests/README.md:267]: This `README` is excellent!

File: goth_tests/README.md
Comment (azawlocki) [inline goth_tests/README.md:53]: Maybe it's worth mentioning here where the test logs are stored by default?

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py
Comment (azawlocki) [inline goth_tests/domain/ya-provider/test_provider_multi_activity.py:39]: Wouldn't it be better to always have a list in `config_overrides`, never `None`?

File: goth_tests/e2e/vm/test_e2e_vm.py
Comment (azawlocki) [inline goth_tests/e2e/vm/test_e2e_vm.py:40]: I think a couple of test cases would benefit if `test_package` were provided by a common fixture.

**Review by azawlocki — APPROVED** [review]
Brilliant! 
I've only left some minor comments, nothing blocking.

**Review by nieznanysprawiciel — APPROVED** [review]

File: goth_tests/README.md
Comment (zakaprov) [reply goth_tests/README.md:38]: Updated in 31cfad0

File: goth_tests/README.md
Comment (zakaprov) [reply goth_tests/README.md:55]: Good catch, added a section on logs in 31cfad0

File: goth_tests/README.md
Comment (zakaprov) [reply goth_tests/README.md]: Updated in 31cfad0

File: goth_tests/README.md
Comment (zakaprov) [reply goth_tests/README.md]: Changed this in 31cfad0

File: goth_tests/domain/ya-provider/test_provider_multi_activity.py
Comment (zakaprov) [reply goth_tests/domain/ya-provider/test_provider_multi_activity.py]: That makes sense, updated in 889aeac7a7772c25eb434e06cf4d63273e8844ec

File: goth_tests/e2e/vm/test_e2e_vm.py
Comment (zakaprov) [reply goth_tests/e2e/vm/test_e2e_vm.py]: I moved them to `helpers/activity.py` instead, since that's where we already take our exe scripts from. This was updated in 09dfb867e168fadc03910d24e1e9230711c5d5fe

**Review by azawlocki — APPROVED** [review]
All my comments have been resolved, thanks!

**Review by pnowosie — DISMISSED** [review]
LGTM!

**Comment by zakaprov** [comment]
Heads up: I'm doing an interactive rebase before merging to get rid of some redundant commits.

File: goth_tests/README.md
Comment (tworec) [inline goth_tests/README.md:47]:
```suggestion
Every `goth` test operates on an isolated, local network of `yagna` nodes which gets created using Docker. Besides the `yagna` containers themselves this network also includes components such as `ya-sb-router` (used for instantiating owned local networking)  or `ganache` local blockchain.
```

File: goth_tests/README.md
Comment (tworec) [inline goth_tests/README.md:54]: I'd mention _"dynamic"_ within point 2. as you did with _"static"_ in 1.

File: goth_tests/README.md
Comment (tworec) [reply goth_tests/README.md:295]: Indeed. Much appreciated effort @zakaprov

Comment (tworec) [reply goth_tests/README.md:295]: ❤️

**Review by tworec — APPROVED** [review]
great job!

**Review by azawlocki — APPROVED** [review]
LGTM, once again!

**Pull request merged**
